storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -357,8 +358,8 @@ bool directory_exists(const char *dir, bool create) {
     }
 
     if (mkdirp(dir, 0777, MYF(0)) < 0) {
-      msg("Can not create directory %s: %s\n", dir,
-          my_strerror(errbuf, sizeof(errbuf), my_errno()));
+      xb::error() << "Can not create directory " << dir << " : "
+                  << my_strerror(errbuf, sizeof(errbuf), my_errno());
```

Comment:
Not a strong opinion though. The previous version did not leave any space between `dir` and `:`. If we want to retain the same then we may have to do
```suggestion
      xb::error() << "Can not create directory " << dir << ": "
                  << my_strerror(errbuf, sizeof(errbuf), my_errno());
```
in all occurences.

Reply:
fixed